Automattic/wp-calypso
# Update ecommerce plan features

Related to pau2Xa-4W0-p2

## Proposed Changes

* This PR updates the feature list copy for the eCommerce plan in all pages that display the plans page. 

### Desktop

![com1](https://github.com/Automattic/wp-calypso/assets/1269602/284f9f80-8b59-42d2-8a1c-6d8073818043)

### Mobile

<img width="309" alt="Screenshot 2023-05-24 at 3 54 36 PM" src="https://github.com/Automattic/wp-calypso/assets/1269602/1b14e4a3-09fe-451c-802f-d9be9b1e5280">


## Testing Instructions



* Verify the plans step in `/start` signup flow has the updated eCommerce features as shown in the description above. a few things to check:
    - verify that the eCommerce plan tagline is updated to reflect the new copy given in pau2Xa-4W0-p2.
    - verify that the feature copy is updated in both the top-of-the-fold plans grid and in the comparison grid below-the-fold.
    - In the comparison grid below-the-fold, verify that all of the new eCommerce features under the Business plan have the text "Available with plugins" copy.
    -  Verify that the tooltip content for all these new features are updated as per the copy provided in the linked P2 post (pau2Xa-4W0-p2).
    - verify the copy in both desktop and mobile matches the screenshots above (and the copy given in the linked P2 post)
    - verify in at least one non-EN locale, in both desktop and mobile, that the new copy is translated.
* Verify the plans step in Calypso's /plans page has the updated eCommerce features as shown in the description above.

## Pre-merge Checklist
